### PR TITLE
DDCE-2706-2-Migrate reactive mongo to hmrc-mongo

### PR DESCRIPTION
**Pull request by anu9**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -33,14 +30,4 @@ object AppDependencies {
 
   val akkaVersion = "2.6.10"
```
Comment (frikit) [inline project/AppDependencies.scala:31]: please delete it as you are not using it

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -33,14 +30,4 @@ object AppDependencies {
...
   val akkaHttpVersion = "10.1.12"
```
Comment (frikit) [inline project/AppDependencies.scala:32]: please delete it as you are not using it

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -5,25 +5,22 @@ object AppDependencies {
 
   val compile = Seq(
     play.sbt.PlayImport.ws,
-    "org.reactivemongo" %% "play2-reactivemongo"            % "0.20.13-play28",
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"             % "0.73.0",
     "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "1.1.0-play-28",
-    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.9.0-play-28",
+    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.12.0-play-28",
     "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "5.24.0",
```
Comment (frikit) [inline project/AppDependencies.scala:11]: still not updated

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -5,25 +5,22 @@ object AppDependencies {
...
-    "com.typesafe.play" %% "play-json-joda"                 % "2.7.4",
     "uk.gov.hmrc"       %% "domain"                         % "6.2.0-play-28"
```
Comment (frikit) [inline project/AppDependencies.scala:12]: still not updated

**Review by frikit — CHANGES_REQUESTED** [review]
Still not done:
- Please fix some scalastyle warnings

File: project/AppDependencies.scala (hunk 3 above)
Comment (anu9) [reply project/AppDependencies.scala:11]: done

File: project/AppDependencies.scala (hunk 2 above)
Comment (anu9) [reply project/AppDependencies.scala:32]: done

File: project/AppDependencies.scala (hunk 4 above)
Comment (anu9) [reply project/AppDependencies.scala:12]: This isn't there in new PR not sure which file you looking at

File: project/AppDependencies.scala (hunk 3 above)
Comment (anu9) [reply project/AppDependencies.scala:11]: done

Comment (frikit) [reply project/AppDependencies.scala:11]: this is not fixed, here is copy paste code from branch:
```
  val compile = Seq(
    play.sbt.PlayImport.ws,
    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"             % "0.73.0",
    "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "1.1.0-play-28",
    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.12.0-play-28",
    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "5.24.0",
    "uk.gov.hmrc"       %% "domain"                         % "6.2.0-play-28"
  )
```

File: project/AppDependencies.scala (hunk 4 above)
Comment (frikit) [reply project/AppDependencies.scala:12]: this is not fixed, here is copy paste code from branch:
```
  val compile = Seq(
    play.sbt.PlayImport.ws,
    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"             % "0.73.0",
    "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "1.1.0-play-28",
    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.12.0-play-28",
    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "5.24.0",
    "uk.gov.hmrc"       %% "domain"                         % "6.2.0-play-28"
  )
```

**Review by frikit — CHANGES_REQUESTED** [review]

File: project/AppDependencies.scala (hunk 3 above)
Comment (frikit) [reply project/AppDependencies.scala:11]: here is code in branch latest version https://github.com/hmrc/maintain-estate-hub-frontend/blob/DDCE-2706-2/project/AppDependencies.scala

File: project/AppDependencies.scala (hunk 4 above)
Comment (frikit) [reply project/AppDependencies.scala:12]: https://github.com/hmrc/maintain-estate-hub-frontend/blob/DDCE-2706-2/project/AppDependencies.scala

**Review by frikit — APPROVED** [review]

**Pull request merged**